sources/web/datalab/polymer/test/settings-manager-test.html

```diff
@@ -20,9 +20,9 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script src="../../web-component-tester/browser.js"></script>
 
-    <script src="../modules/SettingsManager.js"></script>
+    <script src="../modules/settings-manager/settings-manager.js"></script>
```

Comment:
Yes, fixed. The reason it worked was because it wasn't really a Polymer element.